src/script.js

```diff
@@ -0,0 +1,13 @@
+const userManagementLink = document.getElementById(USER_MANAGEMENT_LINK);
+export async function showUserManagementButton() {
+  try {
+    const isSuperUser = await checkUserIsSuperUser();
+    if (isSuperUser) {
+      userManagementLink.classList.remove('element-display-remove');
```

Comment:
1. Can `element-display-remove` be renamed something that talks about the actual behaviour of the class, and not _how_ it's doing this?
2. Maybe we should _add_ this class, when the user is a superuser, to provide additional features, so that most people never see a button flashing away?